product/src/test/e2e/listProductsUseCase.test.js

```diff
@@ -0,0 +1,10 @@
+import request from "supertest";
+
+describe("Lista produtos", () => {
+    it("Lista de produtos", async () => {
+        return await request('http://localhost:3001').get('/products')
```

Comment:
why you are not using app like product/src/test/e2e/createProductUseCase.test.js